dashboard/app/controllers/registrations_controller.rb

```diff
@@ -135,6 +135,15 @@ def create
           metadata: metadata,
         )
       end
+      has_school = current_user.school_info_id?
+      event_metadata = {
+        'has_school' => has_school,
+      }
+      Metrics::Events.log_event(
```

Comment:
I'm fine with having two different events here, but _if_ we wanted to combine them, what if we did something like this

```
# initialize empty variables
metadata, event_name = nil

if Policies::Lti.lti? current_user
  # build up LTI metadata
  metadata = {
    'lms_name' => lms_name,
  }
  # set LTI event_name
  event_name = 'lti_user_created'
} else {
  # build up non-LTI metadata
  event_metadata = {
    'has_school' => current_user.school_info_id?
  }
  # set non-LTI event_name
  event_name = 'Sign Up Finished Backend'
}

# Send event with whatever values we built
Metrics::Events.log_event(
  user: current_user,
  event_name: event_name
  metadata: metadata,
)
```

Reply:
Although I'm realizing now that the default/global 'Sign Up Finished Backend' is probably still desirable for LTI users, so this is probably not what we'd want to do.